app/inventory/barcode/controller.js

```diff
@@ -5,6 +5,8 @@ export default Ember.Controller.extend({
   barcodeUri: function() {
     let id = this.get('model.id');
     let name = this.get('model.name');
+
+    /* eslint new-cap: ['error', { 'capIsNew': false }]  */
```

Comment:
@btecu why is this override needed here?  I'm not seeing a `new` being used here.